src/ios/CDVCamera.m

```diff
@@ -402,6 +444,115 @@ - (NSString*)tempFilePath:(NSString*)extension
     return filePath;
 }
 
+// This is not absolute, but more mightNeedOrientationCorrection
+- (BOOL) needsOrientationCorrection:(UIImage*)image options:(CDVPictureOptions*)options
```

Comment:
Agreed. The only reason this is a function is because I wanted to check:
- if the image really needed an orientation correction (but that is an over optimization),
- if we really wanted to correct gallery image orientation.
Anyway, will be inlined.